src/main/java/Problem3/BookFiction.java

```diff
@@ -25,5 +27,11 @@ public int getLateFeeInDollar() {
     @Override
     public int calcLateFees(int numOfDaysPastDue) {
         // homework
+        if (numOfDaysPastDue > 0 && numOfDaysPastDue < 5) {
+            return(numOfDaysPastDue * lateFeePerDayInDollar);
+        } else if (numOfDaysPastDue >= 5) {
+            return((numOfDaysPastDue * lateFeePerDayInDollar));
```

Comment:
is it just me or both cases are returning the same value? can you make it one case?